src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NoFinalizerCheck.java

```diff
@@ -45,6 +45,22 @@
  * <pre>
  * &lt;module name=&quot;NoFinalizer&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ *  public class Test {
+ *
+ *      // Violation
+ *      protected void finalize() throws Throwable {
```

Comment:
Please add a second `finalize` method but with parameters, there should not be a violation for it.

Reply:
done